routes/api.js

```diff
@@ -160,6 +161,124 @@ router.ws('/results', (ws, req, next) => {
 
 router.ws('/conversation', conversationHandler);
 
+router.post('/conversation/startRecording', (req, res, next) => {
+    const engine = req.app.engine;
+
+    Promise.resolve().then(() => {
+        return engine.assistant.getConversation();
```

Comment:
Pass a conversation ID here, otherwise you'll be the recording the wrong conversation.
(Even in almond-server you can have multiple conversations, if you connect from a different machine than the one running almond-server, and open multiple tabs)

Reply:
how do I get the current ID?

Reply:
That is a beautiful question. It looks like there is no way to pass the right conversation ID. You'll need to add a new message in routes/conversation.js containing the conversation ID.